openapi.yaml

```diff
@@ -3084,6 +3084,106 @@ paths:
           label: Python
           source:
             $ref: examples/TeamRemoveMember.py
+  '/team/sub_teams/{team_id}':
+    get:
+      tags:
+        - Team
+      summary: 'List Sub Teams'
+      description: 'Provides a paginated list of sub teams that belong to a given team.'
+      operationId: teamSubTeams
```

Comment:
You need to define a `requestBody` using the same pattern as other endpoints. That's also where you'll pass in the default request example that gets populated in the Try it console. 
```
      requestBody:
        required: true
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/TeamSubTeamsRequest'
            examples:
              default_example:
                $ref: '#/components/examples/TeamSubTeamsDefaultExample'
          multipart/form-data:
            schema:
              $ref: '#/components/schemas/TeamSubTeamsRequest'
            examples:
              default_example:
                $ref: '#/components/examples/TeamSubTeamsDefaultExample'
```